webapp/channels/src/components/commercial_support_modal/commercial_support_modal.scss

```diff
@@ -8,4 +8,27 @@
     .AlertBanner {
         margin-top: 20px;
     }
+
+    &__packet_contents_download {
+        margin-top: 20px;
+        margin-bottom: 12px;
+    }
+
+    &__options_checkbox_label {
+        padding-left: 12px;
+        font-weight: 500;
+    }
+}
+
+.modal-header .modal-title {
+    font-family: Metropolis, sans-serif;
+    font-weight: 600;
+}
+
+.btn {
```

Comment:
once you've changed the `<span>` tag to an `<i>` tag , you can remove this `.btn` gap rule. It should inherit the right gap from the main button scss